resources/public/api/conf/1.0/lossClaims_amend.raml

```diff
@@ -0,0 +1,25 @@
+is:
+  - acceptJson
+  - contentTypeJson
+  - testScenarioHeader
+  - formatNino
+  - formatClaimId
+  - incorrectOrEmptyBody
+  - formatTypeOfClaim
+  - ruleTypeOfClaim
+  - notFound
+displayName: Amend Loss Claims
+description: This endpoint allows the user to update existing loss claims.
```

Comment:
The corresponding comment for the amend BF losses endpoint is in the singular.

Not in the spec but should we also indicate what it is about the loss claim that can be updated - the typeOfClaim 
 (carry 'direction')?

Reply:
I'm happy to make it singular (it makes more sense to me) - followed the requirement spec though and that has the endpoint down as "Amend Loss Claims" rather than "Amend a Loss Claim" for some reason. Should we change the entire endpoint to be singular?